client/jetpack-connect/style.scss

```diff
@@ -686,3 +686,29 @@
 		padding-right: 319px;
 	}
 }
+
+.jetpack-connect__plan-info {
+	margin-top: 35px;
+	text-align: center;
+	width: 93%;
+	padding-bottom: 30px;
+
+	@include breakpoint( '>480px' ) {
+		width: 100%;
+	}
+
+	@include breakpoint( '>960px' ) {
+		width: 93%;
+	}
+}
+
+.jetpack-connect__plan-info-illustration {
+	padding-top: 30px;
```

Comment:
This spacing looks a bit too much for my eye.

Also, should we apply it to the parent container? Because that way if you switch the elements and prepend something before the animation, the spacing will remain properly at the top of the card. This would also make it consistent with how we declare bottom spacing in the card.

Reply:
Re the amount of padding: design request :)

Reply:
Great, but let's move it to the card. Doesn't make sense to be applied to the illustration IMHO.